std-bits/table/src/main/java/org/enso/table/excel/ExcelRange.java

```diff
@@ -324,6 +325,18 @@ public int getRowCount() {
     return isWholeColumn() ? Integer.MAX_VALUE : bottomRow - topRow + 1;
   }
 
+  public int getLastRow(ExcelSheet sheet) {
```

Comment:
I think it may be worth adding a comment explaining what this method is supposed to do.

Also maybe `getLastNonEmptyRow` will make the intent more clear? Looking at the code it seems to do that - find the last _non empty_ row in the range. Just `getLastRow` suggest last row in the range regardless of its emptiness.